sympy/functions/special/delta_functions.py

```diff
@@ -31,6 +31,20 @@ class DiracDelta(Function):
 
     5) ``DiracDelta(x,k) = 0``, for all ``x != 0``
 
+    Examples
```

Comment:
Thank Aaron. That's very helpful. @sampadsaha5 Please use this text to improve your docstrings.